docs/src/pages/[platform]/connected-components/authenticator/headless/headless-usage.react-native.mdx

```diff
@@ -0,0 +1,137 @@
+import { useRouter } from 'next/router';
```

Comment:
Please review carefully because this file is cloned from `headless-usage.web.mdx`